### DISCUSSION TITLE
Mediatek wifi

### DISCUSSION THREAD

**Discussion by big-mak** [chris1111/My-Github-Pages#4] [Q&A]
Hey Chris, really appreciate your work. I was looking at [this repo](https://github.com/chris1111/D-LinkUtility-Package) and my mediatek chipset (MT7922) is not in the list of those the driver supports. How could I go about adding support for this chipset?

**Comment by chris1111** [comment]
MediaTek Wi-Fi 6E MT7922 is a Wifi 6 not support on [Mediatek macOS Wifi](https://github.com/chris1111/D-LinkUtility-Package)
There is no source exist for the Mediatek macOS Drivers
